### PR TITLE
docs: re-do a bunch of the docs

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

**Comment by jquense** [comment]
prepping for a proper v2 release

File: www/src/components/Anchor.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class Anchor extends React.Component {
+  static propTypes = {
+    id: PropTypes.string,
+  };
+
+  render() {
+    let id =
```
Comment (taion) [inline www/src/components/Anchor.js]: const?

why isn't this a fn component

File: www/src/pages/index.js [hunk 2]
```diff
@@ -1,246 +1,5 @@
...
-            </p>
-
-            <Playground codeText={TransitionSource} />
-          </section>
-        </main>
-      </div>
-    );
-  }
+function App() {
+  return null;
```
Comment (taion) [inline www/src/pages/index.js]: ?

Comment (taion) [reply www/src/pages/index.js]: what's the point of this?

Comment (jquense) [reply www/src/pages/index.js]: it still gets rendered when running locally sometimes, be we redirect immediately

File: www/src/components/Anchor.js (hunk 1 above)
Comment (jquense) [reply www/src/components/Anchor.js]: I just copy/pasted it from index.js

**Pull request merged**